pyproject.toml

```diff
@@ -33,5 +33,8 @@ dependencies = [
 "pytz = 2018.7",
 "requests = 2.20.0",
 "six = 1.11.0",
-"transitfeed = 1.2.16"
+"transitfeed = 1.2.16",
+"typer[all] = 0.9.0",
+"chalky==1.0.0",
+"halo==0.0.29"
```

Comment:
we probably shouldn't necessarily pin the versions so strictly, just to be compatible with what users will commonly have installed. I'd imagine this package would most often get installed to system python so it
's globally available out-of-the-box and then it could conflict with e.g. `requests` which is basically almost a std lib.

it's better to leave some room, e.g. `requests >= 2.22.0, < 3.0.0`, which is like saying we're allowing all requests versions up to the next major, as that likely breaks backwards compatibility (as per semantic versioning)

Reply:
We should also define the dependencies for dev/testing, e.g. `[project.optional-dependencies]`, see https://peps.python.org/pep-0631/#example.